framework/src/utils/MooseUtils.C

```diff
@@ -247,6 +254,15 @@ hasExtension(const std::string & filename, std::string ext, bool strip_exodus_ex
     return false;
 }
 
+void
+rmAll(const std::string & path)
+{
+  auto cmd = std::string("rm -rf ") + path;
+  auto ret = system(cmd.c_str());
```

Comment:
You really should include `cstdlib` and call `std::system` instead. Without an explicit include, we might run into problems as we ship this code around to different systems.

Reply:
Good idea.